core/new-gui/src/app/home/component/home.component.spec.ts

```diff
@@ -13,7 +14,7 @@ describe("HomeComponent", () => {
     TestBed.configureTestingModule({
       declarations: [HomeComponent],
       providers: [{ provide: UserService, useClass: StubUserService }, GoogleService],
-      imports: [RouterTestingModule.withRoutes([{ path: "home", component: HomeComponent }])],
+      imports: [HttpClientModule, RouterTestingModule.withRoutes([{ path: "home", component: HomeComponent }])],
```

Comment:
use `HttpClientTestingModule` instead. the `HttpClientModule` is not to be used in the tests.

Reply:
That's what I was looking for. I forgot the module name.